unit-tests/configuration_test.py

```diff
@@ -43,11 +44,18 @@ def pipe_string(request: FixtureRequest) \
                  {'style.only-multi-rules': {'rules':
                                              'teapot-rule, cheese-rule'}},
                  {'style.plus-rules': {'rules': 'bar-rule',
-                                       'second': 'other thing'}}))
+                                       'second': 'other thing'}},
+                 {'style.arg-rule': {'rules': 'arg-rule(this)'}},
+                 {'style.args-rule': {'rules': 'args-rule(this, that)'}},
+                 {'style.args-rules': {
+                     'rules': 'beef-rule(dee, dum), fish_rule(sam, del)'}}))
 def style_file(request: FixtureRequest) -> Mapping[str, Mapping[str, str]]:
     return request.param
 
 
+_RULE_PATTERN = re.compile(r'[^,(]+(?:\(.+?\))?')
+
+
 class TestConfiguration():
```

Comment:
You've removed the (unnecessary?) brackets from the `class` below - remove these too for consistency?